### PR TITLE
Use sq-config file from PR rather than master

### PR DESCRIPTION
**Pull request by krzyzacy**
ref #1654

### CODE REVIEW

**Comment by krzyzacy** [comment]
@spxtr hope I am doing the right thing with bazel

File: testgrid/config/BUILD [hunk 1]
```diff
@@ -20,6 +20,7 @@ go_test(
     srcs = ["config_test.go"],
     data = [
         "config.yaml",
+        "//mungegithub:submit-queue/deployment/kubernetes/configmap.yaml",
```
Comment (spxtr) [inline testgrid/config/BUILD]: I'm surprised that this isn't private. If it works then it works :)

File: testgrid/config/config_test.go [hunk 2]
```diff
@@ -156,42 +155,35 @@ func TestConfig(t *testing.T) {
 		}
 	}
 
-	configURL := "https://raw.githubusercontent.com/kubernetes/contrib/master/mungegithub/submit-queue/deployment/kubernetes/configmap.yaml"
-	response, err := http.Get(configURL)
+	sqConfigURL := "../../mungegithub/submit-queue/deployment/kubernetes/configmap.yaml"
```
Comment (spxtr) [inline testgrid/config/config_test.go]: It's not a URL anymore.

File: testgrid/config/config_test.go [hunk 3]
```diff
@@ -156,42 +155,35 @@ func TestConfig(t *testing.T) {
...
+				found = true
+				sqJobPool = append(sqJobPool[:i], sqJobPool[i+1:]...)
+				break
 			}
 		}
 
-		for _, testgridJob := range sqJobPool {
-			t.Errorf("Err : testgrid job %v not found in SQ config\n", testgridJob)
+		if !found {
+			t.Errorf("Err : %v not found in testgrid config\n", sqJob)
```
Comment (spxtr) [inline testgrid/config/config_test.go]: There's no need for `\n` in `t.Errorf`, here and elsewhere.

File: testgrid/config/config_test.go (hunk 2 above)
Comment (krzyzacy) [inline testgrid/config/config_test.go]: done.

File: testgrid/config/config_test.go (hunk 3 above)
Comment (krzyzacy) [inline testgrid/config/config_test.go]: done, good to know :-)

**Comment by krzyzacy** [comment]
I would assume green means it works.. PTAL

**Pull request merged**